schema/project-level/project-schema.json

```
@@ -352,6 +352,17 @@
           "codelist": "environmentalGoal.csv",
           "openCodelist": false,
           "minItems": 1
+        },
+        "conservationMeasures": {
+          "title": "Conservation measures",
+          "description": "The measures adopted by the project to protect and enhance biodiversity.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConservationMeasure"
+          },
+          "uniqueItems": true,
+          "minItems": 1,
+          "wholeListMerge": true
```

I think we'll need to remove this, depending on the resolution of https://github.com/open-contracting/infrastructure/pull/434#discussion_r1401173099

```suggestion
          "minItems": 1
```